jscomp/test/PartialApplicationNoRuntimeCurry.js

```diff
@@ -11,7 +12,7 @@ function add(x) {
 function f(u) {
   let f$1 = add(u);
   return function (extra) {
-    return f$1(1, extra);
+    return Curry._2(f$1, 1, extra);
```

Comment:
Now a few Curry calls are back, e.g. here

Reply:
This PR is on top of https://github.com/rescript-lang/rescript-compiler/pull/6893, which does in fact introduce curry.
The next PR has the task of removing them.